corehq/apps/accounting/models.py

```diff
@@ -241,6 +241,7 @@ class BillingAccountAdmin(models.Model):
     domain = models.CharField(
         max_length=256, null=True, blank=True, db_index=True
     )
+    last_modified = models.DateTimeField(auto_now=True, default=datetime.datetime.utcnow())
```

Comment:
I think default should be `datetime.datetime.utcnow` (without calling the function). this will default to the time at which the class is loaded which may not be right.

Also, I'm not clear why you're using both `auto_now` and `default` - is that not redundant?